spec/rubocop/cop/style/empty_line_between_defs_spec.rb

```diff
@@ -268,4 +290,114 @@
       expect(cop.offenses.map(&:line)).to eq([3, 5])
     end
   end
+
+  context 'when a maximum of empty lines is specified' do
+    let(:cop_config) { { 'MinimumEmptyLines' => 0, 'MaximumEmptyLines' => 1 } }
```

Comment:
I do wonder if we should allow 0 as a valid configuration value, but I guess I can live with this.